pages/matchs/new/teams/index.tsx

```diff
@@ -0,0 +1,68 @@
+
+import { Button, Grid, GridItem } from '@chakra-ui/react';
+import type { NextPage } from 'next';
+import { getSession, useSession } from "next-auth/react";
+import { useRouter } from 'next/router';
+import styles from './teams.module.css';
+import Header from '../../../../components/match/new/header';
+import StepTeams from '../../../../components/match/new/step-teams';
+
+const NewMatchStepTeams: NextPage = () => {
```

Comment:
The routes in Next.js works dependeing of the path in the pages folder.
In this moment, to reach this page we need to go to: `/matchs/new/teams`.
Let's keep it simple
`/new-match`